### ISSUE TITLE
General Accessibility Issue with Non-descriptive Button Labels in Hoppscotch

### ISSUE THREAD

**Issue by selametaydogdu** [hoppscotch/.github#1]
Several buttons on the Hoppscotch platform have the `aria-label` attribute set to "button". This poses an accessibility issue, as screen readers only read out "button" when interacting with these buttons, giving the user no context or indication of the button's function.

Example:
<button aria-label="button" ... >...<div class="truncate max-w-54">Sample text</div>...</button>

Suggested Change:
Each button should have a descriptive `aria-label` that matches its actual purpose. For instance, the button with the text "Select Environment" should have the `aria-label` "Select Environment":

<button aria-label="Select Environment" ... >...<div class="truncate max-w-54">Select Environment</div>...</button>
